library/ortools/cc/deps.bzl

```diff
@@ -16,51 +16,87 @@
 #
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository", "new_git_repository")
 
-def build_file_content(lib):
-    return """
-cc_import(
-   name = "lib",
-   shared_library = "lib/{0}",
-   visibility = ["//visibility:public"]
-)
-cc_library(
-  name = "incl",
-  hdrs = glob([
-      "include/ortools/**/*.h",
-      "include/absl/**/*.h",
-      "include/absl/**/*.inc",
-      "include/google/protobuf/**/*.inc",
-      "include/google/protobuf/**/*.h",
-  ]),
-  strip_include_prefix = "include/",
-  visibility = ["//visibility:public"]
-)
-""".format(lib)
+def google_or_tools():
```

Comment:
`or_tools` and all of its dependencies. We get this from the template OR-tools bazel project: https://github.com/or-tools/bazel_or-tools